gfx-glyph/examples/varied.rs

```diff
@@ -153,7 +153,7 @@ fn main() -> Result<(), Box<dyn Error>> {
                                 },
                             },
                             Text {
-                                text: "Eu facilisi maluisset eos.",
+                                text: "Eu facilities maluisset eos.",
```

Comment:
This one just isn't english to begin with

```suggestion
                                text: "Eu facilisi maluisset eos.",
```